### ISSUE TITLE
[BUDI-7198] Accessibility Page Header

### ISSUE THREAD

**Issue by perezstudio** [Budibase/budibase#11034]
When loading different pages inside of budibase, every page header is the title of the app instead of providing page titles which would follow accessibility guidelines.

<sub>[BUDI-7198](https://linear.app/budibase/issue/BUDI-7198/accessibility-page-header)</sub>

**Comment by kcarrandale** [comment]
**Additional info:** the `<title>` should change when navigating the app. Right now it stays the same, which is an accessibility issue because pages must have unique titles.

https://a11y-guidelines.orange.com/en/articles/single-page-app/#update-page-title

Ideally the title would be something like `<title>H1 content/page name - name of app</title>`, with the unique information (page name) coming first. 

(The larger issue is that page updates are not announced by screen readers, which this doesn't solve, but that's something for another, more complex bug.)
